examples/set_avatar.py

```diff
@@ -0,0 +1,74 @@
+########################################################################
+# File name: set_avatar.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+import asyncio
+
+import aioxmpp
+import aioxmpp.avatar
+
+from framework import Example, exec_example
+
+
+class Avatar(Example):
+    def prepare_argparse(self):
+        super().prepare_argparse()
+
+        self.argparse.add_argument(
+            "avatar_file",
+            help="the file PNG image that will be used as avatar."
+        )
+
+        self.argparse.add_argument(
+            "--wipe-avatar",
+            action="store_true",
+            default=False,
+            help="set the avatar to no avatar when exiting."
+        )
```

Comment:
I feel ``--wipe-avatar`` and setting the avatar should be separate tools *or* at least separate options. It shouldn’t be doing both things in one transaction. If I want to wipe my avatar, I shouldn’t need to find an image which I can use as "last avatar before I erase my avatar".